src/tiered/tiered_cursor.c

```diff
@@ -332,17 +332,16 @@ __curtiered_get_current(
 
 /*
  * __curtiered_compare --
- *     WT_CURSOR->compare implementation for the LSM cursor type.
+ *     WT_CURSOR->compare implementation for the tiered cursor type.
  */
 static int
 __curtiered_compare(WT_CURSOR *a, WT_CURSOR *b, int *cmpp)
 {
-    WT_CURSOR_TIERED *alsm;
+    WT_CURSOR_TIERED *atiered;
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
 
-    /* There's no need to sync with the LSM tree, avoid WT_LSM_ENTER. */
```

Comment:
I removed this comment since I have no idea what it is supposed to refer to.  There hasn't been a WT_LSM_ENTER in the code since (at least) 4.0.